ui/oopsyraidsy/data/00-misc/buffs.js

```diff
@@ -1,3 +1,6 @@
+import NetRegexes from '../../../../resources/netregexes.js';
```

Comment:
Instead of import with a long relative path, what about setting a [path alias](https://webpack.js.org/configuration/resolve/#resolvealias) to shorten it?

Reply:
Oh neat! Unfortunately, a number of tests and utility scripts load files outside of webpack contexts, and so I don't think it's safe to use webpack features like that everywhere since those tests and scripts won't know about it.